calico-cloud_versioned_docs/version-3.17/releases.json

```diff
@@ -1,259 +1,263 @@
 [
   {
-    "title": "master",
+    "title": "v3.17.1",
     "tigera-operator": {
       "image": "tigera/operator",
-      "version": "master",
+      "version": "v1.30.3",
       "registry": "quay.io"
     },
     "calico": {
-      "minor_version": "v3.24",
+      "minor_version": "v3.25",
```

Comment:
I think this should be calico v3.26, this is what Enterprise v3.17.1 is based on

Reply:
CE 3.17.1 went out with v3.25. @rene-dekker @danudey did we miss this update for CE?